config/json_exporter/pdm_config.yml

```diff
@@ -119,3 +119,27 @@ metrics:
   path: $.gauges.'rowcount_uc_pdm_model'.value
 - name: rowcount_uc
   path: $.gauges.'rowcount_uc'.value
+- name: state_step_10
```

Comment:
can you please order these properly? like 10,11,12 are before 1,2,3

Reply:
also same for completion time above

- name: completion_min_step_10
  path: $.gauges.'completion_min_step_10'.value
- name: completion_min_step_11
  path: $.gauges.'completion_min_step_11'.value
- name: completion_min_step_12
  path: $.gauges.'completion_min_step_12'.value
- name: completion_min_step_1
  path: $.gauges.'completion_min_step_1'.value
- name: completion_min_step_2
  path: $.gauges.'completion_min_step_2'.value
- name: completion_min_step_3